src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedTypeParameterSymbol.cs

```diff
@@ -77,6 +77,14 @@ public override bool HasReferenceTypeConstraint
             }
         }
 
+        public override bool IsReferenceTypeFromConstraintTypes
+        {
+            get
+            {
+                return _underlyingTypeParameter.IsReferenceTypeFromConstraintTypes || CalculateIsReferenceTypeFromConstraintTypes(ConstraintTypesNoUseSiteDiagnostics);
```

Comment:
> || CalculateIsReferenceTypeFromConstraintTypes(ConstraintTypesNoUseSiteDiagnostics) [](start = 82, length = 84)

nit: similar question here, I didn't understand why we need the extra check (I would have thought that `_underlyingTypeParameter.IsReferenceTypeFromConstraintTypes` would have been sufficient)

Reply:
> similar question here, I didn't understand why we need the extra check (I would have thought that _underlyingTypeParameter.IsReferenceTypeFromConstraintTypes would have been sufficient)

Same reason - the substitution.

---
In reply to: [520175972](https://github.com/dotnet/roslyn/pull/49227#discussion_r520175972) [](ancestors = 520175972)